petastorm/tests/test_spark_dataset_converter.py

```diff
@@ -42,11 +45,11 @@
 from petastorm.spark.spark_dataset_converter import (
     _check_url, _get_horovod_rank_and_size, _get_parent_cache_dir_url,
     _check_rank_and_size_consistent_with_horovod, _make_sub_dir_url,
-    register_delete_dir_handler, _wait_file_available)
+    register_delete_dir_handler, _wait_file_available, _check_dataset_file_median_size,
```

Comment:
nit: I usually use`isort` (a tool installed from pip) to sort the imports in a source file so that it follows the PEP 8 standard.